src/test/java/com/puppycrawl/tools/checkstyle/grammar/GeneratedJavaTokenTypesTest.java

```diff
@@ -701,6 +701,9 @@ public void testTokenNumbering() {
         assertWithMessage(message)
              .that(JavaLanguageLexer.PATTERN_DEF)
              .isEqualTo(213);
+        assertWithMessage(message)
+             .that(JavaLanguageLexer.LITERAL_WHEN)
+             .isEqualTo(216);
```

Comment:
What happened with 214 - 215?

Reply:
Fixed. From not adding the token to the tokens block, it threw the ordering off. Added a test to prevent this in the future.